library/components/TFiltersSection.vue

```diff
@@ -172,16 +179,27 @@ export default {
       }
       return urlFilters;
     },
-    removeFilter(urlFilters) {
-      for (let filter of urlFilters) {
-        this.deleteFilter({ name: filter });
-        if (
-          Object.prototype.hasOwnProperty.call(this.selectedFilters, filter)
-        ) {
-          this.$delete(this.selectedFilters, filter);
+    removeFilter(urlFilters, vfname) {
```

Comment:
What does `vfname` mean?

Reply:
Got it, visibleFilterName I guess... We should be either more explicit or find a better naming, WDYT?

Reply:
Agreed. Updated to visibleFilterName